liberty-maven-plugin/src/it/appsdirectory-notset-configured-it/pom.xml

```diff
@@ -77,6 +82,22 @@
           <serverXmlFile>src/test/resources/server.xml</serverXmlFile>
           <deployPackages>project</deployPackages>
           <looseApplication>false</looseApplication>
+          <copyDependencies>
+            <dependency>
+                <location>lib/global/derby</location>
+                <filter>org.apache.derby:derby</filter>
+            </dependency>
+            <dependency>
+                <filter>commons-logging</filter>
+            </dependency>
+            <dependency>
+                <filter>org.apache.derby:derbytools</filter>
+            </dependency>
+            <dependency>
+                <filter>org.apache.derby:derbyclient:10.15.2.0</filter>
+            </dependency>
+          </copyDependencies>
+
```

Comment:
Maybe add one more IT and cover the default `location` in `copyDependencies/location` ?  Doesn't seem covered anywhere.

Reply:
You are correct that particular case is not covered. I could not cover all three scenarios in one IT, but I guess I can add another one.

Reply:
Thanks.